### PR TITLE
[new release] kittyimg (0.1)

### PR DESCRIPTION
**Pull request by Armael**
An implementation of Kitty's terminal graphics protocol

- Project page: <a href="https://github.com/Armael/ocaml-kittyimg">https://github.com/Armael/ocaml-kittyimg</a>

##### CHANGES:

- initial release

### CODE REVIEW

File: packages/kittyimg/kittyimg.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+license: "MIT"
+homepage: "https://github.com/Armael/ocaml-kittyimg"
+bug-reports: "https://github.com/Armael/ocaml-kittyimg/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.0"}
+  "base64" {>= "3.0.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/kittyimg/kittyimg.0.1/opam:16]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/kittyimg/kittyimg.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+description:
+  "Kitty's terminal graphics protocol allows displaying images in compatible terminal emulators"
+maintainer: ["Armaël Guéneau <armael.gueneau@ens-lyon.org>"]
+authors: ["Armaël Guéneau <armael.gueneau@ens-lyon.org>"]
+license: "MIT"
+homepage: "https://github.com/Armael/ocaml-kittyimg"
+bug-reports: "https://github.com/Armael/ocaml-kittyimg/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.7" & >= "2.0"}
```
[the remark this answers is not in the document]
Comment (Armael) [reply packages/kittyimg/kittyimg.0.1/opam]: fixed

File: packages/kittyimg/kittyimg.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/Armael/ocaml-kittyimg.git"
```
Comment (mseri) [inline packages/kittyimg/kittyimg.0.1/opam:30]:
```suggestion
available: arch != "arm32" & arch != "x86_32"
dev-repo: "git+https://github.com/Armael/ocaml-kittyimg.git"
```
Due to
```

#=== ERROR while compiling kittyimg.0.1 =======================================#
# context              2.2.0~alpha~dev | linux/arm32 | ocaml-base-compiler.4.14.1 | pinned(https://github.com/Armael/ocaml-kittyimg/releases/download/0.1/kittyimg-0.1.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/kittyimg.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p kittyimg -j 79 @install
# exit-code            1
# env-file             ~/.opam/log/kittyimg-7-884704.env
# output-file          ~/.opam/log/kittyimg-7-884704.out
### output ###
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.kittyimg.objs/byte -I /home/opam/.opam/4.14/lib/base64 -intf-suffix .ml -no-alias-deps -o lib/.kittyimg.objs/byte/kittyimg.cmo -c -impl lib/kittyimg.ml)
# File "lib/kittyimg.ml", line 28, characters 26-36:
# 28 |     assert (i > 0 && i <= 4294967295);
#                                ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlopt.opt -w -40 -g -I lib/.kittyimg.objs/byte -I lib/.kittyimg.objs/native -I /home/opam/.opam/4.14/lib/base64 -intf-suffix .ml -no-alias-deps -o lib/.kittyimg.objs/native/kittyimg.cmx -c -impl lib/kittyimg.ml)
# File "lib/kittyimg.ml", line 28, characters 26-36:
# 28 |     assert (i > 0 && i <= 4294967295);
#                                ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```

Comment (Armael) [reply packages/kittyimg/kittyimg.0.1/opam:31]: oof indeed, thanks. Fixed.

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by Armael** [comment]
Thanks :-)

**Pull request merged**